cli/cook/__main__.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
```

Comment:
Should this be under a different directory than `cook`?  `cook_scheduler`?

Reply:
Why do you prefer `cook_scheduler`?

Reply:
Meh.  Let's just leave it cook.  This isn't a library anyway.